docs/sources/reference/components/otelcol/otelcol.receiver.influxdb.md

```diff
@@ -0,0 +1,176 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.receiver.influxdb/
+description: Learn about otelcol.receiver.influxdb
+title: otelcol.receiver.influxdb
+---
+
+# otelcol.receiver.influxdb
+
+{{< docs/shared lookup="stability/experimental.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+`otelcol.receiver.influxdb` receives InfluxDB metrics, converts them into OpenTelemetry (OTEL) format, and forwards them to other otelcol.* components over the network.
+
+You can specify multiple `otelcol.receiver.influxdb` components by giving them different labels.
+
+## Usage
+
+```alloy
+otelcol.receiver.influxdb "influxdb_metrics" {
+  endpoint = "localhost:8086"  // InfluxDB metrics ingestion endpoint
+
+  output {
+    metrics = [...]
+  }
+}
+```
+
+## Arguments
+
+`otelcol.receiver.influxdb` supports the following arguments:
+
+| Name                 | Type           | Description                                                      | Default            | Required |
+| -------------------- | -------------- | ---------------------------------------------------------------- | ------------------ | -------- |
+| `endpoint`           | `string`       | `host:port` to listen for traffic on.                            | `"localhost:8086"` | no       |
```

Comment:
you are missing these arguments that come from the HTTPServerArguments config: max_request_body_size, include_metadata and compression_algorithms. They must be documented here

Reply:
i had to pull out compression bc the otel influxdb reciever doesnt have this compatibility, i included the others tho baked into HTTPServerArguments - take a peek and lmk your thoughts !

Reply:
I think that it does support it? (see my other comment below) I will resolve this comment to continue the discussion on the other one